doc/languages-frameworks/java.section.md

```diff
@@ -1,6 +1,6 @@
 # Java {#sec-language-java}
 
-Ant-based Java packages are typically built from source as follows:
+Java packages are typically built from source as follows:
```

Comment:
Why did you remove "Ant-based"? This section in the document is confusingly titled as "Java" and not "Ant". Removing "Ant-based" will make it more confusing.

Reply:
I wrote this a long time ago 😅 I think I'm going to rewrite the example to manually invoke javac and jar instead:

```bash
javac com/example/MyProject/*.java
jar cf JarExample.jar com/example/MyProject/*.class
```

Reply:
I've replaced the Ant parts of the example with a custom `buildPhase` that invokes `javac` and `jar`, and added a comment above that `buildPhase` to make sure people see the tip about the builders and hooks.